sympy/utilities/autowrap.py

```diff
@@ -138,13 +138,13 @@ def wrap_code(self, routine, helpers=[]):
         oldwork = os.getcwd()
         os.chdir(workdir)
         try:
-            sys.path.append(workdir)
+            sys.path.append(os.getcwd())
```

Comment:
From line 135 it seems that the temporary directory will be either a user specified directory or a temporary directory in the OS's proper temporary space. This change breaks that functionality. Can't  you just type:

```python
autowrap(expr, tempdir=os.getcwd())
```

if you want the temporary directory to be the current working directory?